clarity_epp.py

```diff
@@ -232,18 +262,24 @@ def placement_complete_step(args):
     parser_export_ped.add_argument('process_id', help='Clarity lims process id')
     parser_export_ped.set_defaults(func=export_ped_file)
 
-    parser_export_removed_samples = subparser_export.add_parser('removed_samples', help='Export removed sampels table.', parents=[output_parser])
+    parser_export_removed_samples = subparser_export.add_parser(
+        'removed_samples', help='Export removed sampels table.', parents=[output_parser]
```

Comment:
more of a nitpick, some help's end with a dot ".", others don't, 
help='Create hamilton samplesheets'
help='Export updated illumina samplesheet.'
help='Create tapestation samplesheets'

Reply:
Nice, removed the dots since the default generated messages from argparse also do not end with a dot.